config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1091,7 +1091,7 @@ presubmits:
         - --gcp-zone=us-west1-b
         - --parallelism=1
         - '--label-filter=Feature: containsAny CPUManager && !Flaky'
-        - --skip-regex=\[Flaky\]|\[Slow\]
+        - --skip-regex=""
```

Comment:
Do you really want to allow slow tests in a presubmit?

Flaky is covered in `--label-filter`, slow isn't.

Reply:
> Do you really want to allow slow tests in a presubmit?

Yes. How else will people test their features on a PR?

I can see your argument for a required presubmit but I think people should be able to trigger presubmits even if they are considered "slow" or flaky?

Reply:
I hadn't looked that closely. Yes, for some job which doesn't always run it's fine.